workshop/exercise-1/README.md

```diff
@@ -34,40 +29,46 @@ You must already have a [cluster created](https://console.bluemix.net/docs/conta
     4. uk-south
     5. us-east
     6. us-south
-    Enter a number> 5
+    Enter a number> 6
     ```
-    
+
 6. Install the Kubernetes CLI. Go to the [Kubernetes page](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-binary-via-curl) to install the CLI and follow the steps.
 
 
+> To proceed, your Kubernetes cluster must be started - the state should change from "Pending" to "Normal". Here are the steps to access your cluster. If your cluster is taking too long to create, ask a room moderator for help.
+
+
```

Comment:
are you asking users to create the cluster?  if so, would be good to have it as an optional step/exercise.

Reply:
Can point to the Kube101 stuff for that if we need to say something

Reply:
In ours we do an exercise-0 that covers account setup and cluster creation. https://github.com/nibalizer/istio101/tree/master/workshop/exercise-0

This repo covers a couple different use cases 1) provided accounts, 2) trial accounts created in the workshop, 3) self-study. Hard to handle it all.

Reply:
I know it is not easy :). but handling the scenarios will be much easier than having different forks and branches.  We basically need an optional exercise-0 like you had (or point to k8s101 like @duglin suggested), e.g. if no cluster, let's create it.

Reply:
I referenced kube101